content/W2017/W17-event-ctf.md

```diff
@@ -8,10 +8,12 @@ Summary: Learn about security and networking!
 
 Are you interested in cyber-security and networking, but don't know where to
 start?
+
 [Have you ever wondered what exactly a "CTF" is?](https://ctftime.org/ctf-wtf/)
 Do you want to learn how you too can become a Hacker^TM? Come out to WiCS
-Undergrad's Security and Networking CTF (Capture the Flag)! There will be
-teams and competition and questions about security, crypto, networking and
+Undergrad's Security and Networking CTF (Capture the Flag) hosted in 
+collaboration with [CrySP](https://crysp.uwaterloo.ca/)! There will be teams
+and competition and questions about security, crypto, networking and
```

Comment:
"This is a team competition with questions about security, crypto, networking and infrastructure! There will also be snacks and prizes!